composer.json

```diff
@@ -14,10 +14,10 @@
     "prefer-stable": true,
     "require": {
         "php": ">=5.3.2",
-        "symfony/framework-bundle": "~2.3"
+        "symfony/framework-bundle": "~2.3|~3.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "~3.7.0",
+        "phpunit/phpunit": "~3.7.0|~5.4.6",
```

Comment:
I'm thinking if this supports `~5.4.6` of PHPUnit, maybe we should just bump up the requirement fully and remove the need for support of `~3.7.0`, thoughts @kieljohn ?